examples/containers/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { dispatch, composeStores } from 'redux';
+import CounterApp from './CounterApp';
+import TodoApp from './TodoApp';
+import * as stores from '../stores/index';
+
+@dispatch(composeStores(stores))
```

Comment:
`composeStores` will turn a `stores` exported object into a normal Store with `state, action => state` signature that would combine them. It's a good shortcut, and you may replace it with a custom mechanism like a single Immutable Map if you want to. cc @fisherwebdev